example_dot_env

```diff
@@ -7,7 +7,7 @@ SESSION_SECRET=some_secret
 
 # settings for Central data API. Ask @mattstauffer in staywoke slack for a dev
 # token. see also: https://github.com/StayWokeOrg/central
-DATA_API=stay-woke-central.mattstauffer.co
+DATA_API=http://stay-woke-central.mattstauffer.co
```

Comment:
we'll spec the protocol in the URL since this is already a runtime var. so we don't have to have an environment-based http/https toggle in the code base.